src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-form/rbd-form.component.html

```diff
@@ -95,6 +96,36 @@
         </ng-template>
       </div>
 
+      <!-- Size -->
+      <div class="form-item">
+        <cds-text-label for="size"
+                        i18n
+                        [invalid]="!rbdForm.controls['size'].valid && (rbdForm.controls['size'].dirty)"
+                        [invalidText]="sizeError"
+                        cdRequiredField="Size">Size
+          <input cdsText
+                 type="text"
+                 placeholder="e.g., 10GiB"
+                 id="size"
+                 name="size"
```

Comment:
```suggestion

```
are we using id currently in tests ? If yes, then add otherwise no need

Reply:
https://github.com/ceph/ceph/blob/squid/src/pybind/mgr/dashboard/frontend/cypress/e2e/block/images.po.ts#L22